sci-physics/lhapdf/lhapdf-6.3.0-r2.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2023 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_COMPAT=( python3_{11..12} )
```

Comment:
3.10 we should keep for a bit longer. We drop old implementations in bulk, so there's no need for individual packages to remove old python compats.